app/javascript/packs/validators/circulating_supply.vue

```diff
@@ -42,9 +47,26 @@
       this.update_circulating_supply();
       this.set_continuous_supply_update();
     },
+    mounted: function(){
+      this.$cable.subscribe({
+          channel: "SolPriceChannel",
```

Comment:
Maybe it would be better to create ClusterStatChannel instead of using one that is responsible for other component (sol prices)?

Reply:
I don't want to create too many channels, but I renamed the one for more general and fitting title